proto/cmp/services/info/v1/entry_requirements.proto

```diff
@@ -103,10 +113,18 @@ message LocalizedItemInfo {
   cmp.types.v1.Language language = 3;
 }
 
-// FIXME: We need to clarify what true, false and undefined means for status and
-// maybe update this enum accordingly
+// Status identifies whether an item should be done/provided or rather not. In case
+// of an incident, there might be restrictions.
+//
+// UNKNOWN is different from UNSPECIFIED. For UNKNOWN information about this topic
+// was sought, but it was not possible to find enough information to classify that
+// topic
 enum ItemStatus {
   ITEM_STATUS_UNSPECIFIED = 0;
   ITEM_STATUS_TRUE = 1;
   ITEM_STATUS_FALSE = 2;
+  ITEM_STATUS_CRISIS = 3;
```

Comment:
this is a code change. it's an addition yes, but I think it should be in v2. what do you think ?

Reply:
Agree with that. Every change except comments is a structural change which must be done in a new version if this file is already existing in the c4t branch.